### ISSUE TITLE
Download failed  YT-dlp error:  Exit code: 1

### ISSUE THREAD

**Issue by 01Us** [kylon/Sharedown#74]
Sharedown doesn't work. this is the error after entering credentials:

Failed to send kill signal to download process

Download failed  YT-dlp error:  Exit code: 1

ps. I have Mac M1, the last version of sharedown and updated YT-dlp

**Comment by kylon** [comment]
can you post a log?

**Comment by 01Us** [comment]
> can you post a log?

**Comment by 01Us** [comment]
> can you post a log?

**Comment by kylon** [comment]
enable logs in settings and save
log file should be in ~/Library/Application Support/Sharedown

**Comment by 01Us** [comment]
> enable logs in settings and save log file should be in ~/Library/Application Support/Sharedown

ENABLED LOG ,BUT I HAVE SAME PROBLEMS

**Comment by kylon** [comment]
post the log file

enable log, start the download to get the error, quit the app, get the log file from the path i told you above, remove sensitive data if any, and post it here

**Comment by 01Us** [comment]
> ~/Library/Application Support/Sharedown



> post the log file
> 
> enable log, start the download to get the error, quit the app, get the log file from the path i told you above, remove sensitive data if any, and post it here

I forgot to write it; I followed every step, but there isn't folder Sharedown in Application Support

**Comment by kylon** [comment]
are you sure you are looking at the correct folder?

open a terminal and type `open ~/Library/Application Support/`

**Comment by 01Us** [comment]
> are you sure you are looking at the correct folder?
> 
> open a terminal and type `open ~/Library/Application Support/`

yes, i looked in Library > Application Support >  but doesn't exist  Sharedown's directory 
I also tried from the terminal, but it doesn't exist

**Comment by kylon** [comment]
this is weird

[sharedownt.zip](https://github.com/kylon/Sharedown/files/12743172/sharedownt.zip)

requires nodejs and maybe git too

unpack
open terminal in unpacked folder
type `npm i`
type `npm start`

try again and report back if you see errors in console
